setup.cfg

```
@@ -69,11 +69,11 @@ install_requires =
 
 [options.extras_require]
 pyside2 = 
-    PySide2>=5.12.3,!=5.15.0
+    PySide2>=5.12.3,!=5.15.0, !=5.15.2
```

```suggestion
    PySide2>=5.12.3,!=5.15.0,!=5.15.2
```

Not sure if it makes a difference but I've never seen spaces in these lists